its/plugin/tests/src/test/java/com/sonar/it/php/Tests.java

```diff
@@ -209,6 +209,16 @@ private static void assertAnalyzerLogs(String logs) {
       "org.eclipse.jgit.internal.util.ShutdownHook.cleanup",
       "at java.base/java.lang.Thread.run",
       "org.eclipse.jgit.internal.JGitText",
+      "java.lang.NoClassDefFoundError: ch/qos/logback/classic/spi/ThrowableProxy",
```

Comment:
Did you find any details about it? I wonder how expected it is that scanner can't load logback, and why it only started to occur after your last change.

Reply:
Not really, it's not related to changes on our side but changes on SQ DEV. If we would rerun the CI on the last release we would get the same error that happens on master now (tested locally).
We could test every commit on sonar-enterprise master to see which is responsible, but I'm not sure it's worth the effort.
My plan is to raise it to some SQ Team (don't know which one tbh) like the last time.

Reply:
Makes sense for sure. I just wanted to make sure that it's not like we touched a different part of API and get these errors because of that.

Reply:
No, it was nice thinking. It prompted me to verify my assumptions again.